File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/ArtistSimple.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.os.Parcel;
import android.os.Parcelable;

import java.util.Map;

public class ArtistSimple {
    public Map<String, String> external_urls;
    public String href;
    @ColumnInfo(name = "artist_id")
    public String id;
    public String name;
    public String type;
    public String uri;

    public ArtistSimple() {
    }
}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/ArtistSimple.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.os.Parcel
import android.os.Parcelable

open class ArtistSimple(open val external_urls: Map<String, String> = mapOf(), open val href: String = "", @ColumnInfo(name = "artist_id")open val id: String = "",
                        open val name: String = "", open val type: String = "", open val uri: String = "")
